### ISSUE TITLE
Collection event - verbatim

### ISSUE THREAD

**Issue by OwenLonsdale** [AAFC-BICoE/dina-planning#312]
Requirement: A general "verbatim" field for collection event, or a general "notes" field is needed. 
The type of label I reproduce below as an example is very uncommon, but there are certainly intermediates between this and "typical" labels. Other labels not reproduced here also have references to pages in a notebook, descriptions of behaviour, notes on morphological features, associated loans, and all other manner of unusual text.

Label data:

"Adult Specimen Taxa No. 365.3 Species No. 34,010 (Imago) A number of specimens taken by Robert Harding and one male by Jacob Harding, most near water surface. Brunelle took one male and one female fountaining up high from 1.5-2.5m. Paratypes are specimen numbers 65 (other male), and females 3, 4, 13, and 66. Deposited With To be deposited with NBM under the [sic] Collector Harding R., Harding J., Brunelle, P.M. Collection Canadian National Collection of Insects Storage Envelopes (paratypes separate) Stance Flat Specimens collected Adults All 6 Male 2 Fem 4 Numbers Observed* Adults All 2 Male 2 Fem 3 30 Jun, 95 6.30.1995 Country Canada St./Pr. New Brunswick Sub S/P tt South County Charlotte County MMU Saint James Parish MapBk 84b3 °N 45.38 °W 67.35 SiteCode CNS[745] Elev.Met. 97M Canoose Stream, Highway 745, Canoose river (small) lotic (fast) boulder shoreline; ; rocky bottom, few waterplants; alders, Myrica, mixed forest shoreplants; . *Observed Numbers Codes; 1 = one, 2 = few (2-5), 3 = many (6-25), 4 = great many (>25). P.M. Brunelle, Halifax, Nova Scotia, Canada / CNC 356448."

**Comment by dshorthouse** [comment]
Is this presently accommodated in the CNC db?

**Comment by OwenLonsdale** [comment]
Yes. It's in the CNC DB. We have a "Verbatim" general field for all label data exactly as it appears, but we separate determination label data into a specific field just for that.
Since the CNC DB has both verbatim fields present, we need both fields in DINA as well.
